src/modules/module_23400.c

```diff
@@ -166,15 +166,15 @@ int module_hash_decode (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSE
 
   const u32 iter1 = hc_strtoul ((const char *) iter1_pos, NULL, 10);
 
-  if (iter1 <      1) return (PARSER_SALT_ITERATION);
-  if (iter1 > 999999) return (PARSER_SALT_ITERATION);
+  if (iter1 <     1) return (PARSER_SALT_ITERATION);
+  if (iter1 > 10^10) return (PARSER_SALT_ITERATION);
```

Comment:
`^` is xor operator - take a look here https://en.cppreference.com/w/cpp/keyword/xor.

`iter1 > 10^10` will be compiled into `iter1 > 0` which almost always will be true.